sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -377,12 +374,13 @@ export class CertificateClient {
   public listPropertiesOfCertificates(
     options: ListPropertiesOfCertificatesOptions = {}
   ): PagedAsyncIterableIterator<CertificateProperties> {
-    const requestOptions = operationOptionsToRequestOptionsBase(options);
-
-    const span = createSpan("listPropertiesOfCertificates", requestOptions);
-    const updatedOptions = setParentSpan(span, requestOptions);
-
-    const iter = this.listPropertiesOfCertificatesAll(updatedOptions);
+    const { span, updatedOptions } = createSpan("listPropertiesOfCertificates", options);
+    // TODO: @sadasant - there is a conversion happening here for RequestOptionsBase. In reality
+    // the callers can just avoid the RequestOptionsBase conversion altogether and just pass
+    // OperationOptions instead since (if you trace to the generated client) it would accept it directly.
+    const iter = this.listPropertiesOfCertificatesAll(
```

Comment:
Future TODO: 

TODO: There is a conversion happening here for RequestOptionsBase. In reality
the callers can just avoid the RequestOptionsBase conversion altogether and just pass
OperationOptions instead since (if you trace to the generated client) it would accept it directly.

CC: @sadasant , @maorleger